server.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+var http = require('http');
+
+function sendResponse(res, status, string) {
+  res.writeHead(status, {
+      'Content-Type': 'text/html'
+  });
+  if (string !== '') {
+    res.write(string);
+  }
+  res.end(); //stops browser from waiting for response
+}
+
+function processRoute(req, res) {
+  var writeString = '';
+  //var greet = req.url.match(/^\/greet\/(.+)/);
+
+  if (req.url === '/time') {
+    // Get system time and send back in seconds
+    var time = new Date().getTime();
+    writeString = Math.floor(time/1000).toString();
+    sendResponse(res, 200, writeString);
+
+  } else if ( (req.url === '/greet') && (req.method === 'POST') ) {
+
+    var body = '';
+    req.on('data', function (chunk) {
+      body += chunk.toString();
+    });
+    req.on('end', function() {
+      var greetName = JSON.parse(body);
+      writeString = 'How are you, ' + greetName.name + '?';
+      sendResponse(res, 200, writeString);
+    });
+
+  } else {
+
+    var match = req.url.match(/^\/greet\/(.+)/);
+
+    //console.log("match = " + match[0]);
+    if (match !== null) {
+      if (req.method === 'GET') {
```

Comment:
@dxeware you could combine this conditional and the one from above to make things a bit cleaner...  ex:
if (match !== null && req.method === 'Get')